src/server/auth/server/api_server.go

```diff
@@ -473,8 +473,9 @@ func (a *apiServer) activateInTransaction(ctx context.Context, txCtx *txncontext
 		roleBindings := a.roleBindings.ReadWrite(txCtx.SqlTx)
 		if err := roleBindings.Put(clusterRoleBindingKey, &auth.RoleBinding{
 			Entries: map[string]*auth.Roles{
-				auth.RootUser: {Roles: map[string]bool{auth.ClusterAdminRole: true}},
-				internalUser:  {Roles: map[string]bool{auth.ClusterAdminRole: true}},
+				auth.RootUser:               {Roles: map[string]bool{auth.ClusterAdminRole: true}},
+				internalUser:                {Roles: map[string]bool{auth.ClusterAdminRole: true}},
+				auth.AllClusterUsersSubject: {Roles: map[string]bool{auth.ProjectCreator: true}},
```

Comment:
we should also check if the default project exists, and if it does give `allClusterUsers` ProjectWriter for it